test/common/test_executor.cpp

```diff
@@ -139,9 +134,21 @@ TYPED_TEST(ExecutorInstanceOfAny, executors)
 TEST(ExecutorRuntimeChecks, ExecutorEnviornmentVariableCheck)
 {
   auto env_check = [&](auto exec, const char* env_name) {
-    setenv(env_name, "OFF", true);
+
+#ifdef _WIN32
+    const auto env_string = std::string(env_name) + "=OFF";
+    _putenv(env_string+)
```

Comment:
I'm confused about this syntax. It's a + sign without a second operand.

Reply:
Fixed in aa9a885